### DISCUSSION TITLE
scroll listeners fetch post

### DISCUSSION THREAD

**Discussion by codenashwan** [livewire/livewire#2459] [1. Help]
https://user-images.githubusercontent.com/35005761/106956502-b9a6fe00-6747-11eb-8085-b512c07d1ab3.mp4

hello there , how to stop this empty request

**Comment by PhiloNL** [comment]
You could store keep track of `window.scrollY` for the user and only emit the event if you scroll further down. Something like:

```javascript
if(window.scrollY > userScroll) {
 userScroll = window.scrollY;
 window.livewire.emit('limit');
}
```
